src/i18n/zh-CN/ui.ts

```diff
@@ -1,4 +1,43 @@
 import { UIDictionary } from '../translation-checkers';
 
 export default UIDictionary({
+    'a11y.skipLink': '跳转到内容',
+	'navbar.a11yTitle': '导航顶部',
+    // Site settings
+	'site.title': 'Astro 文档',
+	'site.description': '使用较少的客户端javascript来构建更快的网站。',
+	'site.og.imageSrc': '/default-og-image.png?v=1',
+	'site.og.imageAlt': '星辰浩瀚的太空中的 astro 标志，右侧前景中漂浮着一颗紫色的土星状行星。',
+	// Left Sidebar
+	'leftSidebar.a11yTitle': '站点导航',
+	'leftSidebar.learnTab': '学习',
+	'leftSidebar.referenceTab': '参考',
+	'leftSidebar.noTranslations': '没有找到翻译内容',
+	'leftSidebar.viewInEnglish': '查看英文版本',
+	// Right Sidebar
+	'rightSidebar.a11yTitle': '内容表',
+	'rightSidebar.onThisPage': '在本页内',
+	'rightSidebar.overview': '概述',
+	'rightSidebar.more': '更多',
+	'rightSidebar.editPage': '编辑本页内容',
+	'rightSidebar.translatePage': '翻译本页内容',
+	'rightSidebar.joinCommunity': '加入我们的讨论',
+	// Used in previous/next page links at the bottom of pages
+	'articleNav.nextPage': '下一页',
+	'articleNav.prevPage': '返回',
+	// Used in `<Since>`: Added in: v0.24.0 [NEW]
+	'since.addedIn': '增加到：',
+	'since.new': '新的',
```

Comment:
Here’s a small example. Don’t think we have a version that includes the “new” label at the moment, but it would be after the version number.

| en | zh-CN |
|---|---|
| <img width="219" alt="image" src="https://user-images.githubusercontent.com/357379/167812874-b62b7dc7-eaf7-443f-921a-c93612a79ec1.png"> | <img width="221" alt="image" src="https://user-images.githubusercontent.com/357379/167813124-ef505d04-1fd4-409d-acc7-fa596075060f.png"> |